docs/General/install.rst

```diff
@@ -8,8 +8,23 @@ Freeciv21 has a number of prerequisites.  Note, that apart from the first prereq
 configuration process is smart enough to work out whether your system is suitable. If in doubt, just try it.
 
 An operating system that support Qt
-    Any modern operating system that support Qt is required. As of this writing this is Linux, Microsoft
-    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows Msys2 is supported.
+    Any modern operating system that supports Qt 5.12+ is required. As of this writing this is Linux, Microsoft
+    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows MSYS2 (MingW) is supported.
+
+    Linux Distributions:
+
+    * Arch Current
+    * CentOS 8+
+    * Debian 11+ (Bullseye)
+    * Fedora 28+
+    * Gentoo
+    * KDE neon LTS
+    * Manjaro Stable

```

Comment:
Same with manjaro, I dont think the stable postfix is needed. But better than current I guess, since we wont be worrying too much about supporting the testing branch. (Though it isnt really needed, cuz manjaro's testing is older than arch's stable anyways)

Reply:
I can remove current from Arch and stable from Manjaro if you think that is more accurate.